File: engine/engine-model/src/main/scala/net/server/channel/packet/storage/TakeoutPacket.scala
package net.server.channel.packet.storage

class TakeoutPacket(private var _mode: Byte, private var _type: Byte, private var _slot: Byte) extends BaseStoragePacket(_mode = _mode) {
  def theType: Byte = _type

  def slot: Byte = _slot
}

File: engine/engine-model/src/main/java/net/server/channel/packet/storage/TakeoutPacket.java
package net.server.channel.packet.storage;

public class TakeoutPacket extends BaseStoragePacket {
   private final Byte theType;

   private final Byte slot;

   public TakeoutPacket(Byte mode, Byte theType, Byte slot) {
      super(mode);
      this.theType = theType;
      this.slot = slot;
   }

   public Byte theType() {
      return theType;
   }

   public Byte slot() {
      return slot;
   }
}
